### PR TITLE
Feature - Fullstory

### PR DESCRIPTION
**Pull request by Bagaric**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- Google Tag Manager -->
```
Comment (Kuzirashi) [inline app/index.html]: Maybe we should do this only for production environment somehow?

Comment (Bagaric) [reply app/index.html]: True. Let me check if we can handle this with webpack.

Comment (Bagaric) [reply app/index.html]: Chris added exceptions to localhost & 127.0.0.1 in the google dashboard, so I guess it should be fine now.

Comment (Kuzirashi) [reply app/index.html]: I'm not sold on this but LGTM. :P

**Pull request merged**